### DISCUSSION TITLE
Creep Challenge Idea

### DISCUSSION THREAD

**Discussion by JWally** [abrahamjuliot/creepjs#220] [Ideas]
It would be cool if you could set up a super simple site where this happened:

The challenge is to register your bitcoin wallet 50x in a day at this site. 
The default of the site is to allow one entry per day.
The first person to do this gets the private key of a "prize wallet" that has $50 or whatever in it.
Every month $50 gets added to the pot.
People can donate to the wallet so the site gets more attention.
Each month, the winner can put their name on a wall or something if they want.

No idea how long the wallet would last, but seems like a pretty cheap way to get a TON of information (and attention if you wanted it).

Honestly, it would be cool to see an idea like this take off for both commercial red-teams ("our bot service took down x,y,and z in 45 minutes") and blue-teams ("our service lasted 8 months before it was cracked, giving us valuable insights...")

**Comment by abrahamjuliot** [comment]
That does sound cool. I would do it if the prize was a large bag of fresh organic coffee beans, and the challenge would be to purchase the coffee from our store. By default, the store is locked and gets unlocked after it is convinced a large amount of real web traffic is visiting the site.

For some reason, we get too much traffic. I think if we offered coffee, we would get more humans.
